src/aiperf/records/inference_result_parser.py

```
@@ -509,11 +519,55 @@ async def _compute_server_token_counts(
+                output_texts, _ = self._parse_output_and_reasoning_texts(responses)
+                if output_texts:
+                    tokenizer = await self.get_tokenizer(request_record.model_name)
+                    re_encoded = (
+                        await self._compute_token_count(tokenizer, output_texts) or 0
+                    )
+                    delta = output_token_count - re_encoded
+                    if delta:
+                        self._session_isl_corrections[session_id] = (
+                            self._session_isl_corrections.get(session_id, 0) + delta
+                        )
+            except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)